examples/realworld/ext/article/components/ArticleViewPage.js

```diff
@@ -105,11 +105,7 @@ const ArticleViewPage = (props) => {
         </Grid>
 
         <Grid item xs={8}>
-          <p>
-            <Typography variant="h5">
-              <ReactMarkdown children={article.markdownContent} />
-            </Typography>
-          </p>
+            <ReactMarkdown children={article.markdownContent} />
```

Comment:
Removing the parents fixed most of the _invalid HTML errors_ without affecting the app's appearance all that much.

Reply:
Ah yes, it is me who for some reason keeps thinking that I can put anything in `p`!